src/components/DayPickerNavigation.jsx

```diff
@@ -95,13 +95,21 @@ function DayPickerNavigation({
     );
   }
 
+  const isDefaultNav = isDefaultNavNext || (!isVerticalScrollable && isDefaultNavPrev);
```

Comment:
this might be more clear as
```js
const isDefaultNav = isVerticalScrollable ? isDefaultNavNext : (isDefaultNavNext || isDefaultNavPrev);
```
(i found the negation confusing)